docs/devel/releasing.md

```diff
@@ -144,7 +144,7 @@ Decide what version you're cutting and export it:
 Then, run
 
 ```console
-./release/cut-official-release.sh "${VER}" "${GITHASH}"
+./release/cut-official-release.sh "${VER}" "${GIT_COMMIT_HASH}"
```

Comment:
@thockin  nevermind my previous comment. There are two different usages in the document, one should be "gitcommithash", the exported variable should be "GIT_COMMIT_HASH"